src/app/utils/accountPageUtils.js

```diff
@@ -22,57 +24,162 @@ const makeRequest = (
       if (data.error) updateErrorMessage(data.error);
       return updateAccountHtml(data);
     })
-    .catch(res => console.log('ERROR', res));
+    .catch(res => console.error('ERROR', res));
 };
 
-const addEventListenersToAccountLinks = (
-  links,
+const buildReqBody = (content, itemObj, locationData) => {
+  switch (content) {
+    case 'items':
+      return Object.assign(itemObj, { renewsome: 'YES' });
```

Comment:
When you do `Object.assign`, you're copying `{ renewsome: 'YES' }` into `itemObj`. You should try not to modify parameters. It probably won't result in any bugs but you're better off doing `return { ...itemObj, renewsome: 'Yes' };`